plasmapy/mathematics/tests/test_dispersion.py

```diff
@@ -1,167 +1,200 @@
 """Tests for the plasma dispersion function and its derivative"""
 
+# This file contains experimental usage of unicode characters.
+
 import numpy as np
-from numpy import pi as π
 import pytest
-from astropy import units as u
+from astropy import units
```

Comment:
We should come up with a uniform standard for this... do we want to import as u or as units throughout plasmapy?

Reply:
I like `units` more because it's easier to grep for that rather than for `u`. Then again, that makes the code longer in most places.

Reply:
Oh, I just use `u.` when searching and replacing.

Reply:
Yeah, that works as well. I guess I don't have strong opinions about this.